README.md

```diff
@@ -32,6 +32,12 @@ You can find all CI results recorded on the [![Cypress Dashboard](https://img.sh
 
 If you are looking for BitBucket Pipelines example, check out [bitbucket.org/cypress-io/cypress-example-kitchensink](https://bitbucket.org/cypress-io/cypress-example-kitchensink).
 
+## CI Community Examples
+
+CI | Build status | basic config file | full parallel config
+:--- | :--- | :--- | :---
+IBM Cloud CI | TBD | [Cloud Foundry](https://github.com/iamgollum/cypress-example-kitchensink/tree/281-ibm-cloud-pipeline) | TBD
```

Comment:
I don't think build status or config is that relevant, since some CIs do not have a badge, or don't have single config file. I would just have 2 columns: CI name and link url.

Reply:
but this section is exactly what I wanted - just links to the real-world examples (no need for separate basic or full parallel configs either)